src/zustand/store.js

```diff
@@ -2,6 +2,16 @@ import { create as createZustand } from "zustand";
 
 const useStore = createZustand((set) => ({
   money: 0,
+  tier: [
+    {
+      id: "tier1",
+      isUnlocked: false,
+      isActive: false,
+      name: "sell Wordpress Website",
+      income: 15,
+      delay: 2000,
+    },
+  ],
```

Comment:
Will this `tier` data change over time? It should only be part of state if it is supposed to change

Reply:
Otherwise it could just be a `const`